docs/user_guide.md

```diff
@@ -1105,11 +1105,14 @@ positional arguments:
 
 optional arguments:
   -h, --help            show this help message and exit
-  -s, --suppressed      Filter results to only show suppressed entries.
-                        (default: False)
+  -s, --suppressed      Show only suppressed results instead of only
```

Comment:
Is suppression still a thing?

Reply:
I've just updated the documentation based on the current command output.

Reply:
`CodeChecker cmd results` has a suppressed option

Reply:
I know it does. But does it **need** to have one, since the new API? Note that detection status and review status changes were introduced without a proper command-line touch-up.

Reply:
Ok, I've got your point, this should be rechecked later, as you mentioned.